### ISSUE TITLE
Cannot copy/paste links

### ISSUE THREAD

**Issue by dselman** [accordproject/web-components#114]
Cannot select links in the markdown editor (using mouse or keyboard) because the link editor pops-up during selection.

# Bug Report 🐛
1. Open a document containing a link
2. Attempt to copy/paste the link
3. The link cannot be selected using either the mouse or keyboard because as soon as the selection enters the link the pop up appears

## Expected Behavior
Should be able to select a link.

## Current Behavior
Over-eager pop-up prevents selection.

## Possible Solution
Timer? Click into a link to activate the pop-up? Ensure not in selection mode?

### Desktop
 - OS: Mac
 - Browser: Safari
 - Version: 13.1.1

**Comment by irmerk** [comment]
I see this happening when trying to highlight the link by dragging the curser across it, but note that if you right click to copy it works:

![Jun-16-2020 09-16-49](https://user-images.githubusercontent.com/36460856/84779291-37987800-afb2-11ea-9135-5dd01475d2cb.gif)

**Comment by irmerk** [comment]
I think we should customize the popup behavior to be if the `selection.anchor` is the beginning of the link and `selection.focus` is the end of the link, and no other time?

**Comment by DianaLease** [comment]
> I think we should customize the popup behavior to be if the `selection.anchor` is the beginning of the link and `selection.focus` is the end of the link, and no other time?

I think that is already the case, as I am able to select text that includes more than just the link without it popping up. Only if I select within the link does it pop up. But maybe that's not the exact behavior we want. Maybe we should only popup if the selection is within the link AND collapsed?

**Comment by irmerk** [comment]
Hmm that may have been what I was trying to describe. I'm still not certain what collapsed means in Slate.

**Comment by DianaLease** [comment]
> Hmm that may have been what I was trying to describe. I'm still not certain what collapsed means in Slate.

It basically means the selection is a single point, not a range. So if I click inside the link, my selection is collapsed. If I highlight the link or part of the link, my selection is _not_ collapsed.
